### ISSUE TITLE
Confort plus ne s'affiche pas

### ISSUE THREAD

**Issue by cel57** [Orange-OpenSource/Orange-Confort-plus#164]
Bonjour,
J'ai installé la dernière version de confort plus sur un serveur où est hébergé un site drupal. Confort plus ne s'affiche pas. 
J'ai cette erreur : 
![image](https://github.com/Orange-OpenSource/Orange-Confort-plus/assets/152308513/5209e131-fee0-41c4-9128-3bc558ba6496)

Auriez-vous une idée du problème ?

Merci d'avance.

**Comment by ffoodd** [comment]
Bonjour,

Pourriez-vous préciser quelle version précisément ? Celle sur `main`, ou la dernière _release_ sur GitHub (4.3.0) ?

Vous avez suivi le mode opératoire décrit dans le `README` du dépôt, j'imagine ?

De prime abord, j'ai l'impression qu'il manque un fichier, mais il m'en faudrait un peu plus pour en être sûr.

**Comment by cel57** [comment]
Bonjour,
j'ai pris la version de github. Oui, j'ai suivi le mode opératoire.

**Comment by ffoodd** [comment]
D’après ma première lecture, il est probable que la page qui accueille Confort+ se serve de l’événement `message` d’une manière ou d’une autre, et remplisse par inadvertance la condition qui cherche à récupérer le cookie Confort+. 

Je tenterai de reproduire ce problème et de proposer une solution dès que possible.

**Comment by ffoodd** [comment]
Bonjour @cel57, la version actuellement déployée (donc le code présent sur la branche `main`) fonctionne correctement — il y a donc probablement une interférence quelconque (probablement liée à un `postMessage`).

Est-il possible de consulter l’URL sur laquelle vous insérez Confort+ ?

**Comment by cel57** [comment]
Bonjour,
Je peux vous donner le lien en MP ?

**Comment by ffoodd** [comment]
Oui, bien sûr : éventuellement par mail (vous trouverez une adresse sur mon profil). Sinon dites-moi par quel canal :)

**Comment by ffoodd** [comment]
Comme je l’imaginais, un `postMessage` échoue (généré par la bannière de consentement pour les _cookies_). Je ne peux pas corriger le problème initial, mais je devrais être en mesure de rendre Confort+ plus hermétique aux erreurs externes.

**Comment by ffoodd** [comment]
@cel57 J’ai livré une nouvelle version avec un correctif : je ne peux pas garantir qu’il corrigera votre problème, mais il limite considérablement le risque d’interférence avec d’autres sources de `postMessage`.

Je vous encourage à nous tenir informés si le problème persiste ou si un autre apparait.

**Comment by cel57** [comment]
Bonjour,

J'ai testé la nouvelle version. J'ai toujours des problèmes.
Voici l'erreur que j'ai :
![image](https://github.com/Orange-OpenSource/Orange-Confort-plus/assets/152308513/bfc63f15-072d-4d25-b780-b11e0ad4d33f)

**Comment by cel57** [comment]
Bonjour,
En fait, c'est bon cela fonctionne. Merci.

**Comment by ffoodd** [comment]
Il y avait bien une coquille dans le code (`evenet`), j’ai fait une nouvelle version corrective.
